xml/templates/message_templates/membership_online_receipt_text.tpl

```diff
@@ -1,4 +1,4 @@
-{if $contact.email_greeting}{$contact.email_greeting},{/if}
+{assign var="greeting" value="{contact.email_greeting}"}{if $greeting}{$greeting},{/if}
```

Comment:
I'm confused about what's going on here. Why assign a variable?

Reply:
to avoid a comma if there is none

Reply:
@colemanw note change from {if $contact.email_greeting} to {contact.email_greeting} - it's not a smarty token